app/models/super_good/solidus_taxjar/configuration.rb

```diff
@@ -6,7 +6,15 @@ class Configuration < ::Spree::Base
       include ::Spree::Preferences::Persistable
 
       self.table_name = 'solidus_taxjar_configuration'
-      preference :reporting_enabled, :boolean, default: false
+      preference :reporting_enabled_at_integer, :integer, default: nil
+
+      def preferred_reporting_enabled
+        preferred_reporting_enabled_at_integer.present?
+      end
+
+      def preferred_reporting_enabled_at
+        Time.at(SuperGood::SolidusTaxjar.configuration.preferred_reporting_enabled_at_integer).to_datetime
```

Comment:
We should add some documentation to the configuration in the Readme for this 🙏🏼 

From a UX perspective, using an Epoch based time here seems a bit weird to me, did you consider storing this as a string in a standard datetime format (ISO 8601), i.e. "2022-09-27 18:00:00.000". We would probably want this to be set as UTC time and converted to the Rails configured timezone.